lib/Pass/Pass.cpp

```diff
@@ -277,7 +291,11 @@ struct SouperPass : public ModulePass {
             EC == std::errc::value_too_large) {
           continue;
         } else {
-          report_fatal_error("Unable to query solver: " + EC.message() + "\n");
```

Comment:
I want you to print a strongly worded message here, this can't be silent

Reply:
done